src/views/forms/NewMarketForm/NewMarketSelectionStep.tsx

```diff
@@ -250,71 +250,73 @@ export const NewMarketSelectionStep = ({
+                              assetData: assetToAdd,
+                              clobPairId: nextAvailableClobPairId,
+                              liquidityTier,
+                            })
+                          )
                         )
-                      )
-                    }
-                    tw="[--button-height:auto] [--button-padding:0]"
-                  >
-                    {stringGetter({ key: STRING_KEYS.VIEW_DETAILS })} →
-                  </Button>
-                ),
-              },
-              {
-                key: 'dydx-required',
-                label: (
-                  <span>
-                    {stringGetter({ key: STRING_KEYS.REQUIRED_BALANCE })}{' '}
-                    <Tag>{chainTokenLabel}</Tag>
-                  </span>
-                ),
-                value: (
-                  <div tw="ml-[0.5ch] text-color-text-0">
-                    {stringGetter({
-                      key: STRING_KEYS.OR_MORE,
-                      params: {
-                        NUMBER: (
-                          <Output
-                            useGrouping
-                            type={OutputType.Number}
-                            value={initialDepositAmountBN}
-                            fractionDigits={initialDepositAmountDecimals}
-                            tw="inline-block"
-                          />
-                        ),
-                      },
-                    })}
-                  </div>
-                ),
-              },
-            ].filter(isTruthy)}
+                      }
+                      tw="[--button-height:auto] [--button-padding:0]"
+                    >
+                      {stringGetter({ key: STRING_KEYS.VIEW_DETAILS })} →
+                    </Button>
+                  ),
+                },
+                {
+                  key: 'dydx-required',
+                  label: (
+                    <span>
+                      {stringGetter({ key: STRING_KEYS.REQUIRED_BALANCE })}{' '}
+                      <Tag>{chainTokenLabel}</Tag>
+                    </span>
+                  ),
+                  value: (
+                    <div tw="ml-[0.5ch] text-color-text-0">
+                      {stringGetter({
+                        key: STRING_KEYS.OR_MORE,
+                        params: {
+                          NUMBER: (
+                            <Output
+                              useGrouping
+                              type={OutputType.Number}
+                              value={initialDepositAmountBN}
+                              fractionDigits={initialDepositAmountDecimals}
+                              tw="inline-block"
+                            />
+                          ),
+                        },
+                      })}
+                    </div>
+                  ),
+                },
+              ] satisfies Array<DetailsItem | false | undefined>
```

Comment:
`Array<DetailsItem | false | undefined | null>` is re-used. Maybe make it a re-usable type?